src/main/java/fi/okm/jod/yksilo/controller/ehdotus/TyomahdollisuudetController.java

```diff
@@ -186,6 +185,6 @@ public record LuoEhdotusDto(
 
   @SuppressWarnings("serial")
   static class Response extends ArrayList<Suggestion> {
-    record Suggestion(String name, double score) {}
+    record Suggestion(UUID id, String name, double score) {}
```

Comment:
I think we could drop the name from the suggestion response (not used, and probably dropped from the inference response as well)